sections/plate_notation.tex

```diff
@@ -68,13 +44,13 @@ \subsection{Pachinko Allocation}
 
 \input{figures/dag_structure}
 
-\input{figures/pachinko}
+
+
 
 \subsubsection{Modification to Pachinko Allocation}\label{subsec:mod_pachinko}
-\vejleder[inline]{do you have a comparison in section 5}
-Without modification, \gls{pam} will find topics with the same structure as our taxonomy, but the topics will not actually be based on the taxonomy.\vejleder[inline]{hvad vil det sige?}
+Without modification, \gls{pam} will find topics with the same structure as our taxonomy,  but the taxonomy values would be disregarded during training, since it would generate new taxonomy sequences.
```

Comment:
```suggestion
Without modification, \gls{pam} will find topics with the same structure as our taxonomy, but the taxonomy values would be disregarded during training since it would generate new taxonomy sequences.
```